message AckNewLeader {
  required bytes from = 1;
}